server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/FileWriter.java

```diff
@@ -52,14 +51,13 @@ public final class FileWriter extends DeviceObserver {
 
   private static final long WAIT_INTERVAL_MS = 20;
 
+  private final LocalFileMeta fileMeta;
   private final File file;
```

Comment:
If we store File inside LocalFileMeta, we can remove file here.

Reply:
Yea